addons/events/src/components/Event.tsx

```diff
@@ -98,12 +121,17 @@ class Item extends Component {
     payload: {},
   };
 
-  state = {
+  state: ItemState = {
     isTextAreaShowed: false,
+    failed: false,
+    payload: null,
+    payloadString: '',
+    // eslint-disable-next-line react/no-unused-state,
```

Comment:
I've disabled eslint for this one because of https://github.com/yannickcr/eslint-plugin-react/issues/1759  . That is a pretty old one, maybe there is a better way?